cli/command/swarm/init.go

```diff
@@ -44,21 +47,84 @@ func newInitCommand(dockerCli command.Cli) *cobra.Command {
 	flags.BoolVar(&opts.forceNewCluster, "force-new-cluster", false, "Force create a new cluster from current state")
 	flags.BoolVar(&opts.autolock, flagAutolock, false, "Enable manager autolocking (requiring an unlock key to start a stopped manager)")
 	flags.StringVar(&opts.availability, flagAvailability, "active", `Availability of the node ("active"|"pause"|"drain")`)
+	flags.StringVar(&opts.defaultAddrPool, flagDefaultAddrPool, "", "List of default subnet addresses followed by subnet size (format: <subnet,subnet,..:subnet-size)")
+	flags.SetAnnotation(flagDefaultAddrPool, "version", []string{"1.39"})
 	addSwarmFlags(flags, &opts.swarmOptions)
 	return cmd
 }
 
+// getDefaultAddrPool extracts info from address pool string and fills PoolsOpt struct
+func getDefaultAddrPool(defaultAddrPool string) ([]*net.IPNet, int, error) {
+	var size int
+	var err error
+	var defaultPoolList []*net.IPNet
+	if defaultAddrPool == "" {
+		// defaultAddrPool is not defined
+		return nil, size, nil
```

Comment:
very minor nit:  return 0 explicitly instead of size perhaps to make it clear what it's supposed to be?

Reply:
done